<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>741</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QPushButton" name="Enter">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>132</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nickname">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nickname:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Password">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="chatPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>250</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Send">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Message">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>260</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Group">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CreateGroup">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Join/Create</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Clients">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>81</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Chatbox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Enter</sender>
   <signal>clicked()</signal>
   <receiver>Chat</receiver>
   <slot>on_loginButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Send</sender>
   <signal>clicked()</signal>
   <receiver>Chat</receiver>
   <slot>on_sendButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>427</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_sendButton_clicked()</slot>
  <slot>on_loginButton_clicked()</slot>
 </slots>
</ui>
